apps/studio/data/auth/user-delete-mfa-factors-mutation.ts

```diff
@@ -11,7 +11,11 @@ export type UserDeleteMFAFactorsVariables = {
 }
 
 export async function deleteMFAFactors({ projectRef, userId }: UserDeleteMFAFactorsVariables) {
-  const response = await delete_(`${API_URL}/auth/${projectRef}/users/${userId}/factors`)
+  const response = IS_PLATFORM
+    ? await delete_(`${API_URL}/auth/${projectRef}/users/${userId}/factors`)
```

Comment:
On Supabase Cloud we call to this endpoint: 
`https://api.supabase.io/platform/auth/jnyalqqtavnsmjgplrzt/users/00eb4686-372e-4696-9db0-8a6e0be8294a/factors`
But on local:
`http://localhost:8082/api/auth/default/users/00eb4686-372e-4696-9db0-8a6e0be8294a/factors`
I'm unsure whether this change is correct, and I also have uncertainties regarding the logic on the factors endpoint.